router/core/src/main/scala/io/buoyant/router/RoutingFactory.scala

```diff
@@ -80,7 +80,7 @@ class RoutingFactory[Req, Rsp](
     override def close(d: Time) = conn.close(d)
 
     def apply(req: Req): Future[Rsp] = {
-      Trace.recordBinary("router.label", label)
+      Trace.recordRpc(label)
```

Comment:
tioli: it might be nice to emit both so that it's explicit?  This isn't _really_ the RPC name it's just convenient for zipkin's UI, so this is sort of a hack. it would be nice to reliably be able to look at router.label.